troubleshooting/known_issues/FAS_Loader_Fails.md

```diff
@@ -0,0 +1,67 @@
+## FAS LOADER FAILS
+
+**NOTE:** this procedure is only for csm-0.9.x releases.

```

Comment:
Why is this targeted for CSM 1.0? if its only for 0.9?

Reply:
Will this problem EVER occur on a 1.0 system where the loader is not a job but is part of the API?

Reply:
Agree it should be in the release/0.9 branch - Will remove from this commit and add to 0.9/operations directory.
FAS Loader for 1.0 does rely on the repomd.xml file, so if that file is corrupted FAS Loader will not be able to read from Nexus.  The loader will not need the repomd.xml file to load a single .rpm file.